### ISSUE TITLE
dataset missing?

### ISSUE THREAD

**Issue by KindXiaoming** [thaler-lab/EnergyFlow#30]
When I try to load the dataset using the following line of code

energyflow.qg_nsubs.load(num_data=-1, cache_dir='~/.energyflow')

It says 

raise Exception(error_msg.format(url, e.errno, e.reason))
Exception: URL fetch failure on https://www.dropbox.com/s/y1l6avj5yj7jn9t/QG_nsubs.npz?dl=1: None -- Not Found

**Comment by mattleblanc** [comment]
I am able to download the datasets from zenodo, the dropbox copy does also seem to be missing for me.

```
pfn_example.py	Welcome!
Downloading QG_jets_1.npz from https://www.dropbox.com/s/ztzd1a6lkmgovuy/QG_jets_1.npz?dl=1 to /users/mleblan6/.energyflow/datasets
URL fetch failure on https://www.dropbox.com/s/ztzd1a6lkmgovuy/QG_jets_1.npz?dl=1: None -- Not Found
Failed to download QG_jets_1.npz from source 'dropbox', trying next source...
Downloading QG_jets_1.npz from https://zenodo.org/record/3164691/files/QG_jets_1.npz?download=1 to /users/mleblan6/.energyflow/datasets
Downloading QG_jets_2.npz from https://www.dropbox.com/s/jzgc9e786tbk1m5/QG_jets_2.npz?dl=1 to /users/mleblan6/.energyflow/datasets
URL fetch failure on https://www.dropbox.com/s/jzgc9e786tbk1m5/QG_jets_2.npz?dl=1: None -- Not Found
Failed to download QG_jets_2.npz from source 'dropbox', trying next source...
```

etc.

MLB

**Comment by rikab** [comment]
The dropbox account that previously hosted the data seems to have expired very recently. For all the datasets except qg_nsubs, a backup is hosted on zenodo. 

Will upload a hotfix shortly, switching the default to zenodo. For qg_nsubs, we have a backup on another dropbox, which I will point to.

-Rikab

**Comment by rikab** [comment]
Should be resolved by v1.3.3!

**Comment by KindXiaoming** [comment]
[heart]         Ziming Liu reacted to your message:
________________________________
